path_topic: 'trajectory'
cmd_topic: 'cmd_vel'
global_frame: 'map'
base_frame: 'base_link'
max_v: 0.7
min_v: 0
acc_v: 0.1
max_w: 0.2
acc_w: 0.2
